18F/identity-idp
# Remove nonstandard and redundant icons for government banner

## 🛠 Summary of changes

Removes icon assets to allow design system standard to be used instead:

- Remove `lock.svg` to allow the [design system standard `lock.svg`](https://github.com/uswds/uswds/blob/develop/packages/usa-banner/src/img/lock.svg)
- Remove `icon-dot-gov.svg` to allow the [design system standard `icon-dot-gov.svg`](https://github.com/uswds/uswds/blob/develop/packages/usa-banner/src/img/icon-dot-gov.svg)
- Remove `icon-https.svg` to allow the [design system standard `icon-https.svg`](https://github.com/uswds/uswds/blob/develop/packages/usa-banner/src/img/icon-https.svg)

Visually, they are identical, but the custom lock icon we implemented previously was slightly lighter, which notably stood out more from the surrounding content (see screenshots below).

## 📜 Testing Plan

1. Visit http://localhost:3000
2. Click "Here's how you know"
3. Observe no regressions in appearance of icons in banner content

## 👀 Screenshots

Before|After
---|---
![Screenshot 2024-03-21 at 11 27 02 AM](https://github.com/18F/identity-idp/assets/1779930/8ba0f5d6-fc9a-4f61-af6a-c2ed13ff047c)|![Screenshot 2024-03-21 at 11 26 52 AM](https://github.com/18F/identity-idp/assets/1779930/e8411f2f-9c8f-48e7-b6ce-6b03466a2431)
